tests/step-005.sh

```diff
@@ -15,14 +14,14 @@ cd /tmp/shared/hooks-005.git &&
 if echo "$EXTRA_INSTALL_ARGS" | grep -q "use-core-hookspath"; then
     echo 'n
 y
-/tmp/shared/hooks-005.git
+file:///tmp/shared/hooks-005.git
```

Comment:
Do these go in the `.shared` file? If not, could you please change them back to what they were before?